cob_hardware_config/cob4-3/config/base_joint_limits.yaml

```diff
@@ -0,0 +1,73 @@
+joint_limits:
+  fl_caster_rotation_joint:
+    has_position_limits: false
+    min_position: -3.14
+    max_position: 3.14
+    has_velocity_limits: true
+    max_velocity: 15.0
+    has_acceleration_limits: false
+    max_acceleration: 2.0
+    has_jerk_limits: false
+    max_jerk: 100.0
+    has_effort_limits: false
+    max_effort: 5.0
```

Comment:
@ipa-mdl 
For options that are marked `false` should we remove the respective value?
Is this yaml actually loaded to the parameter server?
Is these parameters actually used within the ros_control/joint_limits_interface part of the driver?